bool is_pick_up_completed
bool is_delivery_completed
time m_time_1
time m_time_2
bool door_status_1
bool door_status_2
int8 task_id
geometry_msgs/PoseStamped goal_1
geometry_msgs/PoseStamped goal_2

